dts/bindings/sensor/nxp,fxas21002.yaml

```diff
@@ -5,8 +5,7 @@ description: FXAS21002 3-axis gyroscope sensor
 
 compatible: "nxp,fxas21002"
 
-include: i2c-device.yaml
-include: spi-device.yaml
+include: [sensor-device.yaml, i2c-device.yaml, spi-device.yaml]
```

Comment:
Please don't include merges in your commit history. When you push an update, please rebase your changes onto the latest version of `main` and resolve conflicts between your changes and `main` within the commit that introduces the conflicts.

Reply:
Thanks, we rebased and fixed the conflicts in their respective commits.